actions/watch.ts

```diff
@@ -26,15 +29,35 @@ export const checkForAndPlaceOrder: ActionFn = async (
         "getTradeableOrder()",
         [Array.from(order)]
       );
-      console.log(`Placing Order: ${order}`);
-      await placeOrder(
-        { ...order, from: contract_address, signature },
-        chainContext.api_url
-      );
-    } catch (e) {
+
+      if (order.validTo * 1000 < new Date().getTime()) {
```

Comment:
Suggestion: I think we could turn this into a variable that describes the condition.

`const orderIsValid = order.validTo * 1000 < new Date().getTime()`

eg.
```suggestion
      if (orderIsValid) {
```

Reply:
I actually like it more when the first condition is valid.

```js
if(orderIsValid)
  valid logic
else
   invalid logic
```

Reply:
fixed